src/App.tsx

```diff
@@ -15,48 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  byAlphabet = 'Sort alphabetically',
+  byLength = 'Sort by length',
```

Comment:
enum's elements should be in CamelCase or UPPER_CASE

Reply:
okey, but now elements are in CamelCase)